src/burnchains/mod.rs

```diff
@@ -387,15 +387,27 @@ impl PoxConstants {
 
     pub fn testnet_default() -> PoxConstants {
         PoxConstants::new(
-            150, // 120 reward slots; 30 prepare-phase slots
-            30,
-            20,
+            50, // 120 reward slots; 30 prepare-phase slots
```

Comment:
That's now 80 reward slots, and 10 prepare-phase blocks?

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2233/commits/dc69365a483107053dc9a776b9998798c8c51578